src/overlays/actors/ovl_Bg_Open_Spot/z_bg_open_spot.h

```diff
@@ -7,7 +7,7 @@ struct BgOpenSpot;
 
 typedef struct BgOpenSpot {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ AnimatedMaterial* animatedMaterial;
+    /* 0x144 */ AnimatedMaterial* animatedMat;
```

Comment:
I had a look in the object, this is actually just 2 texture scrolls, so you can name it texScroll to be more precise.